src/Microsoft.DocAsCode.Build.TableOfContents/TocResolver.cs

```diff
@@ -76,6 +76,9 @@ private TocItemInfo ResolveItemCore(TocItemInfo wrapper, Stack<FileAndType> stac
                     Logger.LogWarning($"Homepage is deprecated in TOC. Homepage {item.Homepage} is overwritten with topicHref {item.TopicHref}");
                 }
             }
+            // validate href
+            ValidateHref(item, file);
```

Comment:
i don't think it would cost much to call the method again, and besides it is more clear to put it inside the method